contracts/wrapperContracts/WrapConversionRate.sol

```diff
@@ -0,0 +1,186 @@
+pragma solidity 0.4.18;
+
+
+import "../ERC20Interface.sol";
+import "../Withdrawable.sol";
+import "../ConversionRatesInterface.sol";
+import "./WrapperBase.sol";
+
+
+contract WrapConversionRate is WrapperBase {
+
+    ConversionRatesInterface conversionRates;
+
+    //add token parameters
+    ERC20     addTokenToken;
+    uint      addTokenMinimalResolution; // can be roughly 1 cent
+    uint      addTokenMaxPerBlockImbalance; // in twei resolution
+    uint      addTokenMaxTotalImbalance;
+    uint      addTokenDataIndex;
+
+    //set token control info parameters.
+    ERC20[]     tokenInfoTokenList;
+    uint[]      tokenInfoPerBlockImbalance; // in twei resolution
+    uint[]      tokenInfoMaxTotalImbalance;
+    uint        tokenInfoDataIndex;
+
+    //general functions
+    function WrapConversionRate(ConversionRatesInterface _conversionRates, address _admin) public
+        WrapperBase(PermissionGroups(address(_conversionRates)), _admin)
+    {
+        require (_conversionRates != address(0));
+        conversionRates = _conversionRates;
+        addTokenDataIndex = addDataInstance();
+        tokenInfoDataIndex = addDataInstance();
+    }
+
+    function getWrappedContract() public view returns (ConversionRatesInterface _conversionRates) {
+        _conversionRates = conversionRates;
+    }
+
+    // add token functions
```

Comment:
should have /////// in first line?

Reply:
I see ///// as underline. to make the line above stand out.
thus I don't add //// above.